client/components/plans/plan-actions/index.jsx

```diff
@@ -31,6 +32,15 @@ module.exports = React.createClass( {
 		}
 
 		if ( this.siteHasThisPlan() ) {
+			if ( this.props.sitePlan.freeTrial ) {
+				return (
+					<Button
+						onClick={ this.handleAddToCart.bind( null, this.cartItem( { isFreeTrial: false } ), 'button' ) }
```

Comment:
This event handler is the same as the one on the button in `upgradeActions`. What do you think about returning `this.upgradeActions()` here, and moving the condition to display 'Purchase Now' or 'Upgrade Now' into that method?